metrics/file_length.py

```diff
@@ -32,8 +32,8 @@ def _parse(self, file, ast):
             ) + 1  # length is zero indexed - therefore we add 1 at the end
         loc = sum(1 for line in file if line.rstrip()) - count_comments
         if loc > 250:
-            return 1
-        return 0
+            data.append(["LOC", path, -1, -1])
```

Comment:
Probably does not matter, but why is this -1, -1 when all others are 1

Reply:
Since its the whole file that surpasses 250 lines without comments, I didn't think there was a need to clarify what lines but only which file.

Reply:
Makes sense